src/ShellClients/ShellClientsManager.vala

```diff
@@ -152,8 +152,10 @@ public class Gala.ShellClientsManager : Object {
             return;
         }
 
-        make_dock (window);
-
         centered_windows[window] = new CenteredWindow (wm, window);
     }
+
+    public bool is_positioned_window (Meta.Window window) {
+        return (window in centered_windows) || (window in windows);
```

Comment:
if this is a window of a positioned windows, it probably counts as a positioned window too?

Reply:
Added that :)